docker/nginx.conf

```diff
@@ -21,5 +21,13 @@ http {
         location / {
             try_files $uri $uri/ /index.html;
         }
+
+        location ~ \.css {
+            add_header  Content-Type    text/css;
```

Comment:
Is this not covered by `/etc/nginx/mime.types`? On line 7 put: 
```
http {
    include       mime.types;
```
Otherwise you can do
```
types {
  text/css css;
}
```

Reply:
Yes. It loads the standard mime.types from nginx. Although, for some reason nginx return `text/html` as the content type for the app.*.js and css files. This was one of the solutions that we suggested

Reply:
I face this issue only when I am running the docker image on localhost. I will revert this change for now